Bluejay.asm

```diff
@@ -184,6 +184,8 @@ Flags0:					DS	1			; State flags. Reset upon motor_start
 Flag_Startup_Phase			BIT	Flags0.0		; Set when in startup phase
 Flag_Initial_Run_Phase		BIT	Flags0.1		; Set when in initial run phase (or startup phase), before synchronized run is achieved.
 Flag_Motor_Dir_Rev			BIT	Flags0.2		; Set if the current spinning direction is reversed
+Flag_Demag_Notify			BIT Flags0.3		; Set when demag has been detected but still not notified
```

Comment:
Please keep the indentations consistent with the other flags

Reply:
Fixed